<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmNavBar</class>
 <widget class="QWidget" name="frmNavBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>widget</string>
  </property>
  <widget class="NavBar" name="navBarLeft" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>100</width>
     <height>300</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>0</y>
     <width>392</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>7</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>8</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <widget class="NavBar" name="navBar1" native="true"/>
    </item>
    <item>
     <widget class="NavBar" name="navBar2" native="true"/>
    </item>
    <item>
     <widget class="NavBar" name="navBar3" native="true"/>
    </item>
    <item>
     <widget class="NavBar" name="navBar4" native="true"/>
    </item>
    <item>
     <widget class="NavBar" name="navBar5" native="true"/>
    </item>
    <item>
     <widget class="NavBar" name="navBar6" native="true"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NavBar</class>
   <extends>QWidget</extends>
   <header>navbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
